src/styles/blocks.scss/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  width: 8 * $cell-size;
+  padding: 20px;
+  gap: $gap-size;
+  margin: auto;
+
+  &__day {
+    display: flex;
+    position: relative;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid black;
+    background-color: #eee;
+    height: $cell-size;
+    width: $cell-size;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: pink;
+    transform: translateY(-20px);
+    transition: transform $transitionDuration;
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/224533521-87450220-cead-4fc2-9d9f-626f770a14b9.png)

Reply:
please write whether this is an appropriate comment, because I have it
![chrome_fVjToVQEbl](https://user-images.githubusercontent.com/31663269/226116700-4554e2fb-899f-4e7e-bc76-b1df48c63928.png)

Reply:
You have a BAD example now, please fix your code so it looks like a GOOD example
![image](https://user-images.githubusercontent.com/96409563/226165056-097eed94-d4c1-4d0c-b99c-c14c656280fb.png)